client/src/pages/Initial/LandingPage<Sherry>/LandingPage<Sherry>.scss

```diff
@@ -0,0 +1,37 @@
+* {
+  box-sizing: border-box;
+}
+:root {
+  --background-color: rgb(81, 0, 128);
+  --text-color: hsl(53, 89%, 52%);
+}
+body {
+  margin: 0;
+}
+.wrapper {
+  display: grid;
+  place-content: center;
+  background-color: var(--background-color);
+  min-height: 100vh;
+  font-family: 'Oswald', sans-serif;
```

Comment:
font-family is also already defined, we're going to stick with the current fonts that we have set up which is `Inter` you can find it in `App.scss` to ensure that we stick with branding guidelines.

since it's already defined "globally" there is no need to add font classes